### PR TITLE
More binary builds v2

### LINKED ISSUE #536: Add Windows aarch64
Supported in wgpu-native: https://github.com/gfx-rs/wgpu-native/pull/398

### PR DESCRIPTION
**Pull request by almarklein**
* [x] Include Windows aarch64 and 32bit builds, closes #536
* [x] Musllinux would be nice, but still an open issue at wgpu-native https://github.com/gfx-rs/wgpu-native/issues/414
* [x] One build per binary, so easier to configure, and easier to extend in the future.

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -11,70 +22,75 @@ on:
...
+          - platform: windows
+            arch: x86
+            os: windows-latest
+          - platform: macos
+            arch: arm64
+            os: macos-latest
+            testable: true
+          - platform: macos
+            arch: x86_64
+            os: macos-13  # latest Intel MacOS
```
Comment (Korijn) [inline .github/workflows/cd.yml:49]: latest or last? :o

Comment (almarklein) [reply .github/workflows/cd.yml:49]: last :)

**Comment by almarklein** [comment]
Need to update the rules for the protected branch to get match the new names of the builds

**Comment by almarklein** [comment]
BTW, we already build a wheel for win32, but that was probably broken because it included the 64bit lib :)

**Review by Korijn — DISMISSED** [review]
Impressive work! Shame the windows wheel test won't work. You could capture the filename in a variable, if that's the issue?

`filename=$(ls dist/*.whl)`
`pip install $filename`

**Comment by almarklein** [comment]
That did the trick, thanks!

**Review by Korijn — APPROVED** [review]

**Pull request merged**